test/Store.test.js

```diff
@@ -6,33 +6,33 @@ import sinon from 'sinon';
 import { Store } from '../src';
 import { DISPATCH_TYPE, STATE_TYPE } from '../src/constants';
 
-describe('Store', function() {
-  beforeEach(function() {
+describe('Store', function () {
+  beforeEach(function () {
     // Mock chrome.runtime API
     global.chrome = {
       runtime: {
-        connect: function(opts) {
+        connect( /* opts */ ) {
```

Comment:
What I meant by this is we should keep the args commented out in the redux dispatch stuff but remove here and in the comment below.